third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,90 @@
+# Stubs for decorator (Python 2 and 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+_Caller = TypeVar("_Caller", bound=Callable[..., Any])
```

Comment:
This is never used as a type var below, since it can just be a normal type alias:

```suggestion
_Caller = Callable[..., Any]
```

Or `Callable[..., Any]` could just be used inline.